.github/workflows/build.yml

```diff
@@ -0,0 +1,51 @@
+name: Build and test
+
+on:
+  pull_request:
+    branches: [ main ]
+
```

Comment:
If we want to change up the default shell for all of the run actions, that can be configured once at the root.

```suggestion
defaults:
  run:
    shell: bash
```

Reply:
alright, that's cool